apps/devhub/tests/test_views.py

```diff
@@ -1124,9 +1125,32 @@ def get_step(self):
         return SubmitStep.objects.get(addon=self.get_addon())
 
 
-class TestSubmitStep1(TestSubmitBase):
+class TestAgreement(TestSubmitBase):
+    def test_agreement_first(self):
```

Comment:
This test is a duplicate of [TestSubmitStep1](https://github.com/Nolski/olympia/blob/signing-api-agreement/apps/devhub/tests/test_views.py#L1152) which will just slow down the suite and create extra test code to maintain.

Instead, I'd suggest asserting contractually that the new view you created is calling the `submit()` handler. You can do that with mock objects like this:
````python
import mock
from django import http
...
with mock.patch('devhub.views.submit') as mock_submit:
    mock_submit.return_value = http.HttpResponse('')
    self.client.get(reverse('devhub.api_key_agreement'))
assert mock_submit.called
````
This will guarantee that the view still works by asserting it inherits all the properties of a known-to-work view handler, the `submit()` function